sound/hybrid_font.h

```diff
@@ -300,6 +304,22 @@ class HybridFont : public SaberBase {
   void SB_Force() override { PlayCommon(&SFX_force); }
   void SB_Blast() override { Play(&SFX_blaster, &SFX_blst); }
   void SB_Boot() override { PlayPolyphonic(&SFX_boot); }
+
+  // Blaster effects, auto fire is handled by begin/end lockup
+  void SB_Stun() override { PlayPolyphonic(&SFX_stun); }
+  void SB_Fire() override { PlayPolyphonic(&SFX_blast); }
+  void SB_ClipIn() override { PlayPolyphonic(&SFX_clipin); }
+  void SB_ClipOut() override { PlayPolyphonic(&SFX_clipout); }
+  void SB_Reload() override { PlayPolyphonic(&SFX_reload); }
+  void SB_Mode() override { PlayPolyphonic(&SFX_mode); }
```

Comment:
I suggest making some sort of fallback here in case "mode.wav" is not available.
Maybe use the beeper? (Or maybe talkie?)

Reply:
Yeah I can do that. Honestly I had not figured out all the hex stuff of talkie yet but want to since I would actually like it to "say" what mode it is in along with a mode sound anyway. But I am a little lost on how to use Talkie

Reply:
Added beep. I'll explore talkie when I get a chance